src/contracts/core/AVSDirectory.sol

```diff
@@ -89,9 +92,7 @@ contract AVSDirectory is
 
         // Check that the signature is valid
         EIP1271SignatureUtils.checkSignature_EIP1271(
-            operator,
-            operatorRegistrationDigestHash,
-            operatorSignature.signature
+            operator, operatorRegistrationDigestHash, operatorSignature.signature
         );
```

Comment:
I'm fine with what `forge fmt` has done here but for the record, the actual way we do this in our codebases is that typically methods with 3+ parameters are called using the explicit parameter/field name syntax, e.g:

```solidity
EIP1271SignatureUtils.checkSignature_EIP1271({
  signer: operator,
  digestHash: operatorRegistrationDigestHash,
  signature: operatorSignature.signature
});
```

Reply:
ok i see we are not calling `checkSignature` like this super consistently, so maybe it's fine for this method. i dont feel super strongly about it

Reply:
Also personally prefer multi-line for paramaters when it exceeds line length